<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 3.1.2 (Python 3.6.7 on win32)" generated="20191019 18:01:23.885" rpa="false">
<suite id="s1" name="Recover Login Info" source="C:\Users\Vicky\PycharmProjects\Parabank_Auto\Test_Data_and_OR\Recover Login Info.robot">
<test id="s1-t1" name="Recover Login info">
<kw name="Log To Console" library="BuiltIn">
<doc>Logs the given message to the console.</doc>
<arguments>
<arg>"Recover user info"</arg>
</arguments>
<status status="PASS" starttime="20191019 18:01:24.045" endtime="20191019 18:01:24.047"></status>
</kw>
<kw name="kw_Recover_forgotten_passwd">
<arguments>
<arg>${td_User1_Details}</arg>
</arguments>
<msg timestamp="20191019 18:01:24.050" level="FAIL">No keyword with name 'kw_Recover_forgotten_passwd' found.</msg>
<status status="FAIL" starttime="20191019 18:01:24.050" endtime="20191019 18:01:24.054"></status>
</kw>
<status status="FAIL" starttime="20191019 18:01:24.044" endtime="20191019 18:01:24.055" critical="yes">No keyword with name 'kw_Recover_forgotten_passwd' found.</status>
</test>
<doc>Suite description</doc>
<status status="FAIL" starttime="20191019 18:01:23.893" endtime="20191019 18:01:24.061"></status>
</suite>
<statistics>
<total>
<stat pass="0" fail="1">Critical Tests</stat>
<stat pass="0" fail="1">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" id="s1" name="Recover Login Info">Recover Login Info</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20191019 18:01:24.003" level="ERROR">Error in file 'C:\Users\Vicky\PycharmProjects\Parabank_Auto\Test_Data_and_OR\Recover Login Info.robot': Resource file 'Keywords\ParaBank_Keywords.robot' does not exist.</msg>
<msg timestamp="20191019 18:01:24.019" level="ERROR">Error in file 'C:\Users\Vicky\PycharmProjects\Parabank_Auto\Test_Data_and_OR\Recover Login Info.robot': Resource file 'Test_Data_and_OR\OR_Parabank.robot' does not exist.</msg>
<msg timestamp="20191019 18:01:24.033" level="ERROR">Error in file 'C:\Users\Vicky\PycharmProjects\Parabank_Auto\Test_Data_and_OR\Recover Login Info.robot': Resource file 'Test_Data_and_OR\TestData_ParaBank.robot' does not exist.</msg>
</errors>
</robot>
